data-processing/scripts/commands.py

```diff
@@ -19,12 +20,14 @@
 from entities import glossary_terms  # pylint: disable=unused-import
 from entities import sentences  # pylint: disable=unused-import
 from entities import symbols  # pylint: disable=unused-import
+from entities import sentences_pdf   # pylint: disable=unused-import
 
 from scripts.pipelines import EntityPipeline, entity_pipelines
 
 # Commands for fetching arXiv sources and preparing for entity processing.
 TEX_PREPARATION_COMMANDS: CommandList = [
     FetchArxivSources,
+    FetchArxivPdf,
```

Comment:
Per discussion above we probably want to compile the latex and get the pdf from that instead of downloading it from arxiv.